src/Credentials/AssumeRoleWithWebIdentityCredentialProvider.php

```diff
@@ -95,6 +95,12 @@ public function __invoke()
             while ($result == null) {
                 try {
                     $token = file_get_contents($this->tokenFile);
+                    if (false === $token) {
+                        clearstatcache(true, dirname($this->tokenFile) . "/" . readlink($this->tokenFile));
```

Comment:
I don't know the answer, but is there any way to reasonably test `clearstatcache`'s efficacy here? Not sure if this would work, but maybe find a way to make a `file_get_contents` call for a given test file that returns false, then clear up the issue with the file and run the provider?